NPE with CompletableFuture

Ran into this NPE when opening up another project while a project was already loaded in the workspace, and then selecting "This Window" when prompted.

```
Internal error: java.lang.NullPointerException: Cannot invoke "java.util.concurrent.CompletableFuture.thenRun(java.lang.Runnable)" because "this.initializeFuture" is null

java.util.concurrent.CompletionException: java.lang.NullPointerException: Cannot invoke "java.util.concurrent.CompletableFuture.thenRun(java.lang.Runnable)" because "this.initializeFuture" is null
	at java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:315)
	at java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:320)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1807)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1796)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.NullPointerException: Cannot invoke "java.util.concurrent.CompletableFuture.thenRun(java.lang.Runnable)" because "this.initializeFuture" is null
	at io.openliberty.tools.intellij.lsp4mp.lsp4ij.LanguageServerWrapper.registerCapability(LanguageServerWrapper.java:729)
	at io.openliberty.tools.intellij.lsp4mp.lsp4ij.LanguageClientImpl.lambda$registerCapability$2(LanguageClientImpl.java:92)
	at java.base/java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1804)
	... 6 more
```

--- a/src/main/java/io/openliberty/tools/intellij/lsp4mp/lsp4ij/LanguageServerWrapper.java
+++ b/src/main/java/io/openliberty/tools/intellij/lsp4mp/lsp4ij/LanguageServerWrapper.java
@@ -733,6 +733,14 @@ public class LanguageServerWrapper {
     }
 
     void registerCapability(RegistrationParams params) {
+        if (initializeFuture == null) { // language server is shutting down but registration messages are still arriving
+            if (LOGGER.isDebugEnabled()) {
+                StringBuffer buffer = new StringBuffer();
+                params.getRegistrations().forEach(reg -> { buffer.append(reg.getMethod()).append(","); });
+                LOGGER.info("registerCapability, initializeFuture==null, ignore capabilities " + buffer);
+            }
+            return;
+        }
         initializeFuture.thenRun(() -> {
             params.getRegistrations().forEach(reg -> {
                 if ("workspace/didChangeWorkspaceFolders".equals(reg.getMethod())) { //$NON-NLS-1$

